bedrock/mozorg/templates/mozorg/home/includes/ctd-promo-de.html

```diff
@@ -0,0 +1,61 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% block content %}
+  <section class="ctd-promo">
+
+    <!-- CTD Promo for non-Firefox users -->
+    <div class="ctd-promo-non-fx">
+      {% call split(
+        block_class='mzp-t-split-nospace mzp-l-split-center-on-sm-md t-pocket-promo',
+        theme_class='mzp-c-split-bg',
+        body_class=None,
+        media_class='mzp-l-split-h-center mzp-l-split-v-center',
+        media_after=False,
+        image=resp_img(
+            url='img/home/2023/ctd-onboarding.png',)
+        ) %}
+    <div class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-family">
```

Comment:
This should be the browser logo, not the family logo.

The wordmark currently in Protocol says "Firefox Browser" because we still haven't been told formally and officially to update it, yet we keep getting designs with the browser logo and just "Firefox" as the wordmark, which means we have to do a custom replacement every time... including this time.